dockerfiles/Dockerfile-centos7

```
@@ -36,37 +35,34 @@
 
 ################# Begin code #######
 
-# Get the latest Centos version from dockerhub
-FROM centos:6
+# Get the latest CentOS 7 version from dockerhub
+FROM library/centos:7
 
 MAINTAINER "dev@qpid.apache.org"
 
-# Install all the required packages. Some in this list were picked off from proton's INSTALL.md (https://github.com/apache/qpid-proton/blob/master/INSTALL.md) and the rest are from dispatch (https://github.com/apache/qpid-dispatch/blob/master/README)
-
 # For centos, some packages are found in the epel repo, so first install access to it
 RUN yum -y install epel-release
 
 # now install the rest of the packages
-RUN yum -y install gcc gcc-c++ cmake libuuid-devel openssl-devel python-unittest2 python-setuptools python-argparse cyrus-sasl-devel cyrus-sasl-plain cyrus-sasl-gssapi cyrus-sasl-md5 swig python-devel java-1.8.0-openjdk-devel git make doxygen valgrind emacs libuv libuv-devel libwebsockets-devel && yum clean all -y
+RUN yum -y install gcc gcc-c++ cmake libuuid-devel openssl-devel cyrus-sasl-devel cyrus-sasl-plain cyrus-sasl-gssapi cyrus-sasl-md5 swig python-devel git make doxygen valgrind emacs
```

Probably should restore

    ```suggestion
RUN yum -y install gcc gcc-c++ cmake libuuid-devel openssl-devel cyrus-sasl-devel cyrus-sasl-plain cyrus-sasl-gssapi cyrus-sasl-md5 swig python-devel git make doxygen valgrind emacs libwebsockets-devel && yum clean all -y
```